reactjs-films-homework/src/components/Header/__tests__/__snapshots__/Header.spec.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Header Header snapshot 1`] = `[Function]`;
```

Comment:
ditto

Reply:
done, test refactored, and snapshot fixed